edx_proctoring/templates/proctoring/seq_proctored_practice_exam_submitted.html

```diff
@@ -0,0 +1,53 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have submitted this practice proctored exam
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Practice Proctoring Session: <b> Completed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      As practice exams do not count towards a grade or credit eligibility
```

Comment:
Practice exams do not affect your grade or your credit eligibility. You have completed this practice exam and can continue with your course.
@griffresch Suggestions?